info/roadmap.md

```diff
@@ -11,9 +11,15 @@ feedback we receive from our ~~adoring fans~~ users.
 
 | Feature Ref. | Feature Details                                                                                                                           | Planned For   |
 | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| TF-157       | Chain Groups - ways to slice and dice your chains. | End of 2024 |
-| FA-598       | Initial console integration - control your plugins from a console! | End of 2024 |
-| TF-1659      | Network Redundancy - mirror two **transform**.engines so that they can be used in a redundant configuration.                              | 2025          |
+| FA-598       | Initial console integration - control your plugins from a DiGiCo Quantum console! | March 2025 |
+| TF-157       | Chain Groups - ways to slice and dice your chains. | March 2025 |
+| TF-1094      | Support side-chains | March 2025 |
+| TF-444, TF-445  | Support larger than mono and stereo channel widths. | March 2025 |
+| TF-1109     | Teleport multiple plugins at same time | March 2025 |
+| TF-2454     | High Density/ Overview Chain view | March 2025 |
+| TF-112, TF-137, TF-2204 | Ability to identify DSP core assigned to chain, as well source of DSP overruns. | March 2025 |
+| TF-133      | Tap Tempo synchronisation | March 2025 |
+| TF-1659      | Network Redundancy - mirror two **transform**.engines so that they can be used in a redundant configuration.                              | Q4 2025          |
```

Comment:
```suggestion
| TF-78        | Network Redundancy - mirror two **transform**.engines so that they can be used in a redundant configuration                              | Q4 2025          |
```